src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -115,6 +121,18 @@ public async Task NewTsConfig(CancellationToken cancellationToken)
             }
         }
 
+        public async Task<TSConfigResponse> TsConfigFiles(string tsConfigFilePath, CancellationToken cancellationToken)
+        {
+            var responseString = await httpWrapper.PostAsync(BuildServerUri("tsconfig-files"), tsConfigFilePath, cancellationToken);
+
+            if (string.IsNullOrEmpty(responseString))
```

Comment:
Same error handling as for `AnalyzeJS`. This wrapper will throw if we get an invalid response. The response could contain error codes, but it's up to the caller to handle them.